dockerfiles/evmapp/entrypoint.sh

```diff
@@ -29,30 +29,30 @@ export SCNODE_REMOTE_KEY_MANAGER_ENABLED
 
 
 # Function(s)
+fn_die() {
+  echo -e "$1" >&2
+  sleep 5
+  exit "${2:-1}"
+}
+
 detect_ext_ip() {
   local usage="Detect external IPv(4|6) address - usage: ${FUNCNAME[0]} {(4|6)}}"
   [ "${1:-}" = "usage" ] && echo "${usage}" && return
-  [ "$#" -ne 1 ] && { echo -e "${FUNCNAME[0]} error: function requires exactly one argument.\n\n${usage}"; exit 1;}
+  [ "$#" -ne 1 ] && { fn_die "${FUNCNAME[0]} error: function requires exactly one argument.\n\n${usage}"; }
```

Comment:
I think you do not need `{}` anymore since you are not grouping multiple commands together :shushing_face: